src/main/resources/assets/dimdoors/lang/de_de.json

```diff
@@ -43,5 +43,217 @@
   "item.stabilized_rift_signature.unbound.info": "Erster Klick speichert die Position, \\nweitere Klicks kreieren Spalte, die \\ndie erste und die aktuelle Position verbinden.",
   "item.unstable_dimensional_door.info": "Vorsicht: Führt zu einem zufälligen Ziel",
   "item.oak_dimensional_door.info": "Platziere auf dem Block unterhalb eines \\nSpalts, um ein Portal zu erstellen, \\noder platziere irgendwo in einer \\nkleinen Dimension, um sie zu verlassen.",
-  "entity.dimdoors.monolith": "Monolith"
-}
\ No newline at end of file
+  "entity.dimdoors.monolith": "Monolith",
```

Comment:
You can remove everything above `"entity.dimdoors.monolith": "Monolith",`
We dont use those translations anymore